nabclockd/nabclockd.py

```diff
@@ -11,6 +11,7 @@
 
 class NabClockd(nabservice.NabService):
     DAEMON_PIDFILE = "/run/nabclockd.pid"
+    SKIP_WAKEUP_SOUNDS_SEC = 300
```

Comment:
Do we really need this constant? Can you please explain the logic here and maybe provide a test?

Reply:
It was to avoid the sleep/wake up sounds to be played right after boot, as this was strange and maybe also conflicting with other sounds (i don't remember well), so i put this 5 min delay.